Sources/SKSupport/Logging.swift

```diff
@@ -12,18 +12,52 @@
 
 import Basic
 import Foundation
+import Utility
 
 #if canImport(os)
 import os // os_log
 #endif
 
-public enum LogLevel: Int, Equatable {
+public enum LogLevel: Int, Equatable, CustomStringConvertible, ArgumentKind {
```

Comment:
Style: Please separate conformances to their own extensions. The exception is synthesized conformances like `Equatable` or `Hashable` and conformances that are a core part of what the type is.